### PR TITLE
solved task

### PR DESCRIPTION
**Pull request by AntonKurylo**

### CODE REVIEW

File: src/main/java/mate/academy/hibernate/relations/model/Movie.java [hunk 1]
```diff
@@ -2,10 +2,26 @@
...
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany
+    @JoinTable(name = "movies_actors",
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/hibernate/relations/model/Movie.java:22]: Are you sure about this relation?

File: src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        this.actorDao = new ActorDaoImpl(sessionFactory);
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java:13]: Don`t init actor dao in service layer, do it in Dao layer

File: src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java [hunk 3]
```diff
@@ -1,16 +1,25 @@
...
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElse(null);
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java:23]: orElse null is very bad idea, use orElseThrow

**Review by Sanderfuck — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/hibernate/relations/model/Movie.java (hunk 1 above)
Comment (AntonKurylo) [reply src/main/java/mate/academy/hibernate/relations/model/Movie.java:22]: Why not? There can be several actors in one movie

**Pull request closed without merging**